config/manager/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/openstack-k8s-operators/osp-director-operator
-  newTag: 0.0.1
+￼ newTag: 0.0.1
```

Comment:
is this to change a bad char, or introduce it :)

Reply:
yes, it is a bad char. I think its gone in the latest commit.